tasks/package.py

```diff
@@ -53,7 +53,13 @@ def check_size(ctx, filename: str = 'package_sizes.json', dry_run: bool = False)
             decision = "⚠️ Warning"
         else:
             decision = "✅ Passed"
-        display_message(ctx, ancestor, size_message, decision)
+        # Try to display the message on the PR when a PR exists
+        if os.environ.get("CI_COMMIT_BRANCH"):
+            try:
+                display_message(ctx, ancestor, size_message, decision)
+            # PR commenter asserts on the numbers of PR's, this will raise if there's no PR
+            except AssertionError as exc:
+                print(f"Got `{exc}` while trying to comment on PR, we'll assume that" "this is not a PR.")
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Why do you have a tuple here?
Instead of
```suggestion
                print(f"Got `{exc}` while trying to comment on PR, we'll assume that this is not a PR.")
```

Reply:
The autoformatter, I manually split the string because I thought it was too long and the formatter put it back on the same line.